README.md

```diff
@@ -33,8 +33,9 @@ Run the bash scripts in the following order:
 ## References
 
 [1] R. C. Gessner, C. B. Frederick, F. S. Foster, and P. A. Dayton, “Acoustic Angiography: A New Imaging Modality for Assessing Microvasculature Architecture,” International Journal of Biomedical Imaging, vol. 2013, p. e936593, Jul. 2013, doi: 10.1155/2013/936593.
+
 [2] L. Biewald, “Experiment tracking with weights and biases.” 2020. [Online]. Available: https://www.wandb.com/
 
 ## License
 
-The codes are licensed under the MIT license.
+All the code in this repository are licensed under the MIT license.
```

Comment:
Should this be "all the code in this repository is"?

Reply:
Yes